clang/lib/Frontend/InitPreprocessor.cpp

```diff
@@ -485,6 +485,11 @@ static void InitializeStandardPredefinedMacros(const TargetInfo &TI,
 
     if (LangOpts.SYCLValueFitInMaxInt)
       Builder.defineMacro("__SYCL_ID_QUERIES_FIT_IN_INT__", "1");
+
+    // Set __SYCL_DISABLE_PARALLEL_FOR_RANGE_ROUNDING__ macro for
+    // both host and device compilations if -fintelfpga flag is present.
+    if (LangOpts.AheadOfTimeFPGACompilation)
```

Comment:
Shouldn't this now correlate with the `-fsycl-disable-range-rounding` setting given the new driver changes?

Reply:
The scope that `-fintelfpga` covers is not enough, as it misses the cases when `-fsycl-targets=spir64_fpga` is used.  We should be able to use just `-fsycl-disable-range-rounding` to set the macro as the driver is taking care of determining all of the instances in which it should be set (whenever FPGA is used)

Reply:
Thanks @mdtoguchi for the clarification. I will update patch.